server/src/test/java/org/elasticsearch/action/admin/cluster/configuration/TransportAddVotingConfigExclusionsActionTests.java

```diff
@@ -181,7 +182,7 @@ public void testWithdrawsVotesFromNodesMatchingWildcard() throws InterruptedExce
 
         assertTrue(countDownLatch.await(30, TimeUnit.SECONDS));
         assertThat(clusterService.getClusterApplierService().state().getVotingConfigExclusions(),
-                containsInAnyOrder(otherNode1Exclusion, otherNode2Exclusion));
```

Comment:
Could you revert the whitespace changes to this file?

Reply:
Done.